### PR TITLE
Add getting-started/deploy-app/with-the-cli

### PR DESCRIPTION
**Pull request by Malyuk-A**
Adds the getting-started/deploy-app/with-the-cli page.

Uses the asciinema player described and added in this PR:  https://github.com/nitindermohan/oak-new-docs/pull/8

# Screenshots
image
image

### CODE REVIEW

File: content/docs/getting-started/deploy-app/deploy-cli.md [hunk 1]
```diff
@@ -14,4 +14,147 @@ seo:
...
   noindex: false # false (default) or true
+asciinema: true
 ---
+
+Besides using the API directly or clicking through the dashboard UI, you can also interact with Oakestra using the Oakestra Command Line Interface.
+
+{{< callout context="caution" title="Requirements" icon="outline/alert-triangle">}}
+  - You have a running Oakestra setup.
+  - You can access the APIs at `<IP_OF_CLUSTER_ORCHESTRATOR>:10000`
+  - You have read and familiarized yourself with the [API]({{< relref "deploy-api.md" >}}) and [Dashboard]({{< relref "deploy-dashboard.md" >}}) approaches.
```
Comment (giobart) [inline content/docs/getting-started/deploy-app/deploy-cli.md]: Is this really a requirement? I think that I can use the CLI even if I never used the API and the Dashboard right?

File: content/docs/getting-started/deploy-app/deploy-cli.md [hunk 2]
```diff
@@ -14,4 +14,147 @@ seo:
...
+{{< /callout >}}
+
+
+### Working with Application SLAs
+As detailed in the [API Approach]({{< relref "deploy-api.md" >}}) Oakestra apps and services are defined in SLAs.<br>
+The OAK-CLI comes with a set of pre-defined default SLAs.<br>
+All available SLAs can be inspected via the `oak application sla` command.
+{{< asciinema key="cli_sla_inspect" poster="0:16" >}}
+
+All SLAs are stored in your home directory in the `oak_cli` folder.
```
Comment (giobart) [inline content/docs/getting-started/deploy-app/deploy-cli.md:106]: I would suggest to not flood the user with info, what if we just say `oak_cli/SLAs` and then simply show 

```
~/oak_cli
 ├── SLAs
 │   ├── blank_app_without_services.json
 │   ├── default_app_with_services.json
 │   └── mysla.json
```

omitting the rest of the files such as .init_flag etc.?

File: content/docs/getting-started/deploy-app/deploy-cli.md [hunk 3]
```diff
@@ -14,4 +14,147 @@ seo:
   description: "" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
```
Comment (giobart) [inline content/docs/getting-started/deploy-app/deploy-cli.md]: the flag `toc: true` can be enabled to show the table of content on the right and side. This way a user can skip sections. What do you think?

File: content/docs/getting-started/deploy-app/deploy-cli.md (hunk 1 above)
Comment (Malyuk-A) [reply content/docs/getting-started/deploy-app/deploy-cli.md]: Ok I have removed that line for now - My idea is that it is very beneficial to know about the underlying things the CLI uses and to know about the alternatives - and yes I agree this is not a hard requirement

File: content/docs/getting-started/deploy-app/deploy-cli.md (hunk 2 above)
Comment (Malyuk-A) [reply content/docs/getting-started/deploy-app/deploy-cli.md:106]: Agreed - I will abstract away the other content

File: content/docs/getting-started/deploy-app/deploy-cli.md (hunk 3 above)
Comment (Malyuk-A) [reply content/docs/getting-started/deploy-app/deploy-cli.md]: My bad - was not aware what the flag was doing

**Review by giobart — APPROVED** [review]

**Review by nitindermohan — CHANGES_REQUESTED** [review]
Please see commit: https://github.com/nitindermohan/oak-new-docs/commit/5953d6c8f4d42df914ac8bfebd467272fd14927f

Also updated the high level setup doc to link the CLI support

**Comment by Malyuk-A** [comment]
> Please see commit: [5953d6c](https://github.com/nitindermohan/oak-new-docs/commit/5953d6c8f4d42df914ac8bfebd467272fd14927f)
> 
> Also updated the high level setup doc to link the CLI support

Thank you for committing your preferred changes @nitindermohan 

Is there anything else you would like to have changed? Otherwise, why did you set your review as Request Changes (if you already committed them)? - or is this just for my sake to have a look at the committed changes?

**Comment by nitindermohan** [comment]
Only for your sake to check the commits. I did not have any review comments for your PR so you can ignore it.

**Pull request merged**